packages/cozy-device-helper/src/platform.spec.js

```diff
@@ -0,0 +1,46 @@
+import {
+  isWebApp,
+  isMobileApp,
+  isIOSApp,
+  isAndroidApp,
+  getPlatform,
+  isFlagshipApp
+} from './platform'
+
+describe('platforms', () => {
+  it('should identify is a web application', () => {
+    expect(isWebApp()).toBeTruthy()
+  })
+  it('should identify is a mobile application', () => {
+    window.cordova = true
+    expect(isMobileApp()).toBeTruthy()
+    window.cordova = undefined
+    expect(isMobileApp()).toBeFalsy()
+  })
+  it('should identify is an iOS or Android application', () => {
+    window.cordova = { platformId: 'ios' }
+    expect(isIOSApp()).toBeTruthy()
+    expect(isAndroidApp()).toBeFalsy()
+    window.cordova = { platformId: 'android' }
+    expect(isIOSApp()).toBeFalsy()
+    expect(isAndroidApp()).toBeTruthy()
```

Comment:
very good! THANKS 🎉 

nit: Just one more point. We just notice that the Jest matcher toBeTruthy is not very adapted to check boolean value.
Using toBe(true) is more strict.

I think we should use it everywhere (especially in the moved tests) 🙏

Reply:
I agree but it should be done at a later time, eg when converting the library to TS